holoviews/tests/ui/bokeh/test_callback.py

```diff
@@ -208,7 +218,9 @@ def test_multi_axis_tap_datetime(serve_hv):
 
     def test():
         assert s.xs == {'x': np.datetime64('2024-01-12T13:26:44.819277')}
-        assert s.ys == {'y1': 18.13070539419087, 'y2': 76.551867219917}
+        assert s.xs == {'x': np.datetime64('2024-01-12T13:26:44.819277')}
+        assert 18 <= s.ys["y1"] <= 18.5
+        assert 76 <= s.ys["y2"] <= 76.5
```

Comment:
```suggestion
        assert len(s.ys) == 2
        assert np.isclose(s.ys["y1"], 18.13070539419087)
        assert np.isclose(s.ys["y2"], 76.551867219917)
```

Will this work?

Reply:
No unfortunately that was what I had earlier.

Reply:
https://github.com/holoviz/holoviews/commit/2b90cfc74b60979db76633f288eedfdc08d1b71c

Reply:
You had integers. [`np.isclose`](https://numpy.org/doc/stable/reference/generated/numpy.isclose.html) also set a buffer much lower than you have.

Have you tried it locally?

Reply:
Not sure if integers matter. It works locally, but failed on GitHub

<img width="1675" alt="image" src="https://github.com/user-attachments/assets/b47772ab-ff7b-44a4-82ab-a704b4d1032c">

Reply:
Can you try without rounding down? Before, there were a lot of decimals.

Reply:
Locally, it is set to an int

```
E       TimeoutError: wait_until timed out in 5000 milliseconds: assert False
E        +  where False = <function isclose at 0x1032de320>(18, 18.13)
E        +    where <function isclose at 0x1032de320> = np.isclose
```

---

```python

    def test():
        assert s.xs == {'x': np.datetime64('2024-01-12T13:26:44.819277')}
        assert s.xs == {'x': np.datetime64('2024-01-12T13:26:44.819277')}
        assert np.isclose(s.ys["y1"], 18.13)
        assert np.isclose(s.ys["y2"], 76.55)

```

Reply:
Adding an atol

```python
    def test():
        assert s.xs == {'x': np.datetime64('2024-01-12T13:26:44.819277')}
        assert s.xs == {'x': np.datetime64('2024-01-12T13:26:44.819277')}
        assert np.isclose(s.ys["y1"], 18.2, atol=0.5)
        assert np.isclose(s.ys["y2"], 76.5, atol=0.5)
```